pkg/lib/long-running-process.js

```diff
@@ -91,6 +91,15 @@ export class LongRunningProcess {
         return this.systemdClient.call(O_SD_OBJ, I_SD_MGR, "StopUnit", [this.serviceName, "replace"], { type: "ss" });
     }
 
+    // Should journal log be cleared? -- Separate option?
```

Comment:
This comment does not belong here, the lib file does nothing with the journal. Clearing the journal text box in examples/long-running-process/index.js may be a nice touch -- should happen in the `STOPPED` state transition then. I leave it up to you if you want to do that.